src/CommandingOfficers/Commander.java

```diff
@@ -567,4 +538,30 @@ public int getIncomePerTurn()
     }
     return count * (gameRules.incomePerCity + incomeAdjustment);
   }
+
+  @Override
+  public void applyToUMLImpl(UnitModList uml)
```

Comment:
Didn't realize we had a Unified Markup Language implementation built into our game. CoOl.